integrations/php/install-driver.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+set -o errexit  # Exit the script with error if any of the commands fail
+
+install_composer ()
+{
+    # See: https://getcomposer.org/doc/faqs/how-to-install-composer-programmatically.md
+    EXPECTED_CHECKSUM="$(php -r 'copy("https://composer.github.io/installer.sig", "php://stdout");')"
+    php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
+    ACTUAL_CHECKSUM="$(php -r "echo hash_file('sha384', 'composer-setup.php');")"
+
+    if [ "$EXPECTED_CHECKSUM" != "$ACTUAL_CHECKSUM" ]
+    then
+        >&2 echo 'ERROR: Invalid installer checksum'
+        rm composer-setup.php
+        exit 1
+    fi
+
+    php composer-setup.php --quiet
+    RESULT=$?
+    rm composer-setup.php
+
+    if [ $RESULT -ne 0 ]; then
+        exit $RESULT
+    fi
+}
+
+install_extension ()
+{
+    if [ "x${PHPC_BRANCH}" != "x" ] || [ "x${PHPC_REPO}" != "x" ]; then
+        REPO=${PHPC_REPO:-https://github.com/mongodb/mongo-php-driver}
+        BRANCH=${PHPC_BRANCH:-master}
+
+        echo "Compiling driver branch ${BRANCH} from repository ${REPO}"
+
+        BUILD_DIR="$(mktemp -d)"
+
+        git clone --depth 1 --recurse-submodules --shallow-submodules --branch ${BRANCH} ${REPO} ${BUILD_DIR}
+        pushd ${BUILD_DIR}
+
+        phpize
+        ./configure --enable-mongodb-developer-flags
+        make all -j20 > /dev/null
+        make install
+
+        popd
+    elif [ "x${PHPC_VERSION}" != "x" ]; then
+        echo "Installing driver version ${PHPC_VERSION} from PECL"
+        pecl install -f mongodb-${PHPC_VERSION}
+    else
+        echo "Installing latest driver version from PECL"
+        pecl install -f mongodb
+    fi
+
+    echo "extension=mongodb.so" > ${PHP_PATH}/lib/php.ini
+}
+
+# TODO: Remove this after https://jira.mongodb.org/browse/BUILD-13026 is resolved
+if [ -d "/opt/php/${PHP_VERSION}-64bit/bin" ]; then
+    PHP_PATH="/opt/php/${PHP_VERSION}-64bit/bin"
+else
+    # Try to find the newest version matching our constant
+    PHP_PATH=`find /opt/php/ -maxdepth 1 -type d -name "${PHP_VERSION}*-64bit" -print | sort -V -r | head -1`
+fi
+
+if [ ! -d "$PHP_PATH" ]; then
+    echo "Could not find PHP binaries for version ${PHP_VERSION}. Listing available versions..."
+    ls -1 /opt/php
+    exit 1
+fi
+
+export PATH=${PHP_PATH}/bin:${PATH}
```

Comment:
Nit: this could be extracted to a separate file and included using `source`. It would remove the duplicate usage and make it easier to drop once BUILD-13026 is done.

Reply:
I'm intentionally avoiding adding extra scripts because finding the current path to refer to them is a rabbit hole (https://stackoverflow.com/q/59895). Both `install-driver.sh` and `workload-executor` are invoked from the root directory of `drivers-atlas-testing` and changing the path within breaks `workload-executor` since it expects to write its output files in that directory. `install-driver.sh` doesn't care if we `cd`, but I'd find it irksome if both scripts ended up being inconsistent in this regard.

Reply:
Ah, in that case let's leave this as is and focus on getting the `<major>.<minor>` links in place 👍